symphony/bdk/core/auth/jwt_helper.py

```diff
@@ -1,20 +1,23 @@
 import datetime
 from jose import jwt
 
+DEFAULT_EXPIRATION = 5 * 58
+
 
 def create_signed_jwt(private_key_config, username, expiration=None):
     """Creates a JWT with the provided user name and expiration date, signed with the provided private key.
 
-    :param private_key_config: The private key configuration for a service account or an extension app.
-    :param username: the username of the user to authenticate
-    :param expiration: expiration of the authentication request in seconds.
+    :param private_key_config:  The private key configuration for a service account or an extension app.
+    :param username:            The username of the user to authenticate
+    :param expiration:          Expiration of the authentication request in seconds.
+                                By default the signed jwt will be valid in maximum 290 seconds.
```

Comment:
thanks, just maybe precise that 5 minutes is the maximum accepted by the backend?